docs/shared/recipes/deployment/deploy-nuxt-to-vercel.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Deploying Nuxt applications to Vercel
+description: A detailed guide on how to deploy Nuxt applications from Nx workspaces to Vercel.
+---
+
+# Deploying Nuxt applications to Vercel
+
+## Configure your Nuxt application appropriately for Vercel deployment
+
+In your application's `nuxt.config.ts` file, ensure that the `nitro` property is configured to use the `vercel` preset, and that the `output.dir` property is set to the appropriate directory. The directory that Vercel expects to find the application's build output (as explained in the [Vercel documentation](https://vercel.com/docs/build-output-api/v3)) in is `.vercel/output` at the root of the repository/workspace. So, if your application is located at `apps/my-app` then the `output.dir` property should be set to `../../.vercel/output`.
+
+For example, if your Nx workspace is structured like this:
+
+```treeview
```

Comment:
Normally the treeview is the result of running the `tree .` command. Like

```treeview
├── apps
│   └── demo
│       └── .babelrc
├── libs
│   ├── a
│   │   └── .babelrc
│   └── b
│       └── .babelrc
└── babel.config.json
```

Reply:
oops!